package.json

```diff
@@ -56,8 +56,11 @@
     "normalizr": "^3.2.2",
     "prop-types": "^15.5.8",
     "react": "^15.4.2",
+    "react-async-script": "^0.9.1",
     "react-dom": "^15.4.2",
     "react-flexbox-grid": "^1.1.3",
+    "react-google-recaptcha": "^0.9.5",
+    "react-recaptcha": "^2.3.1",
```

Comment:
I was testing both libraries react-google-recaptcha is better and more stable, but i figured there might be a chance we change between them, If that is not the case then react-recaptcha can be removed